app/models/invenio_rdm_record_converter.rb

```diff
@@ -226,6 +219,18 @@ def build_creator_contributor_json(creator)
     File.write(MEMOIZED_PERSON_OR_ORG_DATA_FILE, @@person_or_org_data)
     # return the actual json
     json
+  end # build_creator_contributor_json
+
+  def additional_titles(titles)
+    additional_titles_size = titles.size-1
+    return nil if additional_titles_size < 0
+    titles.last(additional_titles_size).map{ |title| {"title": title, "type": "alternative_title", "lang": ENG} }
+  end
+
+  def additional_descriptions(descriptions)
```

Comment:
Same as the above comment. The block in that comment can be changed to fit here pretty easily. Actually, if you wanted to abstract a bit you could even write one function that handles both of these in one, but it would probably be more trouble than it's worth.